flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionGraphRestartTest.java

```diff
@@ -97,8 +103,10 @@ private void executeOperationForAllExecutions(
         }
     }
 
-    private SlotPoolImpl createSlotPoolImpl() {
-        return new TestingSlotPoolImpl(TEST_JOB_ID);
+    private SlotPool createSlotPoolImpl() {
+        return new DeclarativeSlotPoolBridgeServiceFactory(SystemClock.getInstance(), DEFAULT_RPC_TIME_OUT, DEFAULT_IDLE_SLOT_TIME_OUT, DEFAULT_BATCH_TIME_OUT).createSlotPoolService(TEST_JOB_ID).castInto(SlotPool.class).orElseThrow(() ->
+                new IllegalStateException(
+                        "The AdaptiveScheduler requires a DeclarativeSlotPool."));
```

Comment:
Can we simply create a `DeclarativeSlotPoolBridge` here?

Reply:
Got it